ts/packages/node/src/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './stores/IniFile';
 export * from './stores/AWSParameterStore';
 export * from './stores/AWSSecretsManager';
 export * from './stores/AzureKeyVault';
+export * from './stores/CloudBees';
```

Comment:
למה לדחוף את זה באמצע

Reply:
Thought the list should be A-Z..
I'll move it to the end